spec/class.dd

```diff
@@ -632,11 +632,10 @@ $(GNAME Destructor):
         object.
         )
 
-        $(P The  program can explicitly inform the garbage collector that an
-        object is no longer referred to (with the delete expression), and
-        then the garbage collector calls the destructor
-        immediately, and adds the object's memory to the free storage.
-        The destructor is guaranteed to never be called twice.
+        $(P The program can explicitly inform the garbage collector that an
+        object is no longer referred to with $(REF1 destroy, object), and
+        then the garbage collector calls the destructor immediately.  The
+        destructor is guaranteed to never be called twice.
```

Comment:
Missing parenthesis here.

Reply:
The parens is there on the next line: 639.